server/index.js

```diff
@@ -28,6 +31,10 @@ app.use("/api/v1/", apiRouter);
 app.use("/", reactRouter);
 app.use(RequestSyntaxErrorHandler);
 
+fs.mkdirSync("files", { recursive: true });
+
+app.use(AuthenticationMiddleware, express.static("files"));
```

Comment:
Style nit: could we group the `app.use()`?

Reply:
yep